jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-uninstall.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- job:
+    name: openshift-cluster-integreatly-uninstall
+    display-name: 'Openshift Cluster Integreatly Uninstall'
+    project-type: pipeline
+    parameters:
+      - string:
+          name: 'clusterName'
+          default: ''
+          description: '[REQUIRED] The name of the target cluster to uninstall Integreatly against'
+      - string:
+          name: 'openshiftMasterUrl'
+          default: ''
+          description: '[REQUIRED] The public URL of the target OpenShift cluster'
+      - string:
+          name: 'clusterAdminUsername'
+          default: 'integreatly'
+          description: '[REQUIRED] Username of the cluster admin account'
+      - string:
+          name: 'clusterAdminPassword'
+          default: 'Password1'
+          description: '[REQUIRED] Password of the cluster admin account'
+      - string:
+          name: 'uninstallerGitUrl'
```

Comment:
I think this should just be installationGitUrl like the other jobs. It's not really because it was in the install job, more that it is the installation repo url.